ivy_tests/test_ivy/test_frontends/test_tensorflow/test_signal.py

```diff
@@ -10,35 +9,45 @@
 
 # hann_window
 
-
-@handle_cmd_line_args
-@given(
+handle_cmd_line_args
+given(
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("numeric"),
-        num_arrays=2,
-        shared_dtype=True,
+        available_dtypes=helpers.get_dtypes("numeric")
```

Comment:
the periodic argument requires a Boolean, but your current generation method produces two numeric values.